test/widgets/login_test.dart

```diff
@@ -61,7 +64,44 @@ void main() {
     expectErrorFromText('email@example.com', ServerUrlValidationError.noUseEmail);
   });
 
-  // TODO test AddAccountPage
```

Comment:
There's lots more to test on this page :slightly_smiling_face: — so this TODO is still relevant and let's keep it.